.github/workflows/benchmark_gpu_core_crypto.yml

```diff
@@ -126,8 +125,7 @@ jobs:
           --commit-date "${{ env.COMMIT_DATE }}" \
           --bench-date "${{ env.BENCH_DATE }}" \
           --name-suffix avx512 \
-          --walk-subdirs \
-          --throughput
+          --walk-subdirs
 
```

Comment:
I don't see the bench-type flag meaning we never measure/parse throughput ?

same everywhere else